api-docs/cloud-dedicated/ref.yml

```diff
@@ -1442,14 +1442,27 @@ paths:
 
         Use this endpoint to send data in [line protocol](/influxdb/cloud-dedicated/reference/syntax/line-protocol/) format to InfluxDB.
 
-        InfluxDB does the following when you send a write request:
+        InfluxDB Cloud Dedicated does the following when you send a write request:
 
-        1. Validates the request
-        2. If successful, attempts to [ingest the data](/influxdb/cloud-dedicated/reference/internals/durability/#data-ingest); _error_ otherwise.
-        3.  If successful, responds with _success_ (HTTP `204` status code), acknowledging that the data is written and queryable; _error_ otherwise.
+        1. Validates the request.
+        2. If successful, attempts to [ingest the data](/influxdb/cloud-dedicated/reference/internals/durability/#data-ingest); [error](/influxdb/cloud-dedicated/write-data/troubleshoot/#review-http-status-codes) otherwise.
+        3.  If successful, responds with an HTTP `201 "Created"` status code, acknowledging that data is written and queryable.
+        If some points in the batch are rejected, the response body contains details about the [rejected points](/influxdb/cloud-dedicated/write-data/troubleshoot/#troubleshoot-rejected-points).
 
-        To ensure that InfluxDB Cloud handles writes in the order you request them,
-        wait for a success response (HTTP `2xx` status code) before you send the next request.
+        To ensure that InfluxDB handles writes in the order you request them,
+        wait for the response before you send the next request.
+
+        #### Write endpoints
+
+        The [`/write`](#operation/GetLegacyQuery) and [`/api/v2/write`](#operation/PostWrite) endpoints are functionally equivalent for writing data to InfluxDB Cloud Dedicated.
+
+        - Use the [`/write` endpoint](#operation/GetLegacyQuery) for InfluxDB v1 parameter compatibility.
+        - Use [`/api/v2/write` endpoint](#operation/PostWrite) for InfluxDB v2 parameter compatibility.
+
+        #### Rate limits
+
+        _Write_ rate limits apply.
+        For more information, see [limits and adjustable quotas](/influxdb/cloud-dedicated/admin/billing/limits/).
```

Comment:
I didn't know we have write rate limits in Dedicated. TIL!

Reply:
> I didn't know we have write rate limits in Dedicated. TIL!

Oops